### DISCUSSION TITLE
I'm not able to figure out how to change the font family while still having to call @chakra-ui/pro-theme

### DISCUSSION THREAD

**Discussion by Cesar3rd** [chakra-ui/chakra-ui#7550] [Q&A] [answered]
Hey friends, I hope you are all safe.

I'm on the last touches of finishing my first beautiful chakra pro project, but I'm facing a minor issue that couldn't figure out.

How can I change the font family of my app while still having to call the @chakra-ui/pro-theme because all of the websites was built with pro chakra components?

would love to hear your experiences and suggestions🤗

**Comment by jmims-magnusdigital** [comment, marked as answer]
This is what I had to do:

```
export const theme({
 ...,
 proTheme
});

theme.fonts = {
  heading: 'whatever',
  body: 'whatever',
};
```
